packages/ragbits-core/src/ragbits/core/config.py

```diff
@@ -19,5 +25,24 @@ class CoreConfig(BaseModel):
         LLMType.STRUCTURED_OUTPUT: "ragbits.core.llms.factory:simple_litellm_structured_output_factory",
     }
 
+    # Path to functions that returns instances of diffrent types of Ragbits objects
```

Comment:
Note: for LLMs `default_llm_factories` still exists as a separate setting - I don't love this but we have special needs for LLM factories, so I haven't come up with a better idea